pkg/apis/app/v2/kieapp_types.go

```diff
@@ -1051,6 +1055,20 @@ type CORSFiltersObject struct {
 	MaxAgeValue *int32 `json:"maxAgeValue,omitempty"`
 }
 
+// StartupStrategy
+type StartupStrategy struct {
+	// StartupStrategy to use. Default is OpenShiftStartupStrategy, option is ControllerBasedStartupStrategy.
+	StrategyName string `json:"strategyName,omitempty"`
+	// Controller Template Cache TTL to use when the ControllerBasedStartupStrategy is choosed and BC is deployed, default is 5000
```

Comment:
Shouldn't be there used OSSS?
```suggestion
	// Controller Template Cache TTL to use when the OpenShiftStartupStrategy is choosed and BC is deployed, default is 5000
```

Reply:
also, you can copy the description from templates or image.yaml